packages/core/src/payment/payment-method-request-sender.ts

```diff
@@ -7,6 +7,10 @@ import {
     SDK_VERSION_HEADERS,
 } from '../common/http-request';
 
+import HeadlessPaymentMethodConfig from './headless-payment-method-config';
+import { HeadlessPaymentMethodResponse } from './headless-payment-method-response';
+import { HeadlessPaymentMethodType } from './headless-payment-method-type';
+import HeadlessPaymentRequestOptions from './headless-payment-request-options';
```

Comment:
Can we group them into a `headless-payment` folder for better orgnization as I can see things are quite different here?

Reply:
Moved to the separate folder